docs/organisation-logos.md

```diff
@@ -0,0 +1,46 @@
+# Organisation logos
+
+## Overview
+
+The [Organisation logo component](https://components.publishing.service.gov.uk/component-guide/organisation_logo) can be used to render organisation logos that use either the [Government Identity System](https://gcs.civilservice.gov.uk/guidance/marketing/branding-guidelines/), such as [HMRC](https://components.publishing.service.gov.uk/component-guide/organisation_logo/hm_revenue_customs/preview), or have a [brand exemption](https://gcs.civilservice.gov.uk/guidance/marketing/branding-guidelines/#Brand-exemption-process), such as [HMPS](https://components.publishing.service.gov.uk/component-guide/organisation_logo/hm_prison_service/preview).
+
+This documentation provides details on how to update and add new logos that are part of the Government Identity System.
+
+## Images required
+
+The four images below are required for the organisation logo component. The image format required is a transparent PNG with the whitespace trimmed.
```

Comment:
It seems relatively easy to identify where images need to be added or updated but I wonder whether it would still be helpful to mention that that images should be added to `app/assets/images/govuk_publishing_components/crests`.

Reply:
Yeah, I think it's always best to be as clear as possible as well, I've added this in, thanks